src/Endpoint.sol

```diff
@@ -15,4 +16,9 @@ abstract contract Endpoint {
         pure
         virtual
         returns (EndpointStructs.EndpointMessage memory endpointMessage);
+
+    /// @notice pause the endpoint
+    function _pauseEndpoint() internal {
+        _pause();
+    }
```

Comment:
Do we need this function? It's effectively just a pass-through to `_pause` since it's also internal.
Endpoint implementations such as `EndpointStandalone` can call directly into `Pausable._pause` instead of calling `_pauseEndpoint`.